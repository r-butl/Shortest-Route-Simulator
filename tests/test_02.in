4 4 9 9
0 3
0 0
1 0
2 1
3 0
0 1 5
0 2 8
1 3 5
2 3 8
