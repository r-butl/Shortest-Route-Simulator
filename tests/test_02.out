Verify Path: 1
16: 0 2 3 
